airbyte-integrations/connectors/source-azure-blob-storage/pyproject.toml

```diff
@@ -16,9 +16,9 @@ repository = "https://github.com/airbytehq/airbyte"
 include = "source_azure_blob_storage"
 
 [tool.poetry.dependencies]
-python = "^3.9,<3.12"
-pytz = "==2024.1"
-airbyte-cdk = {extras = ["file-based"], version = "^1"}
+python = "^3.11,<3.12"
+pytz = "^2024.1"
+airbyte-cdk = {extras = ["file-based"], version = "^4"}
```

Comment:
It's possible that a higher version will work without problems, but I wanted to go to a minimal version that works with Python 3.11